documentation/src/main/asciidoc/chapter-3-defining-a-mapper.asciidoc

```diff
@@ -532,17 +532,22 @@ In case you want to disable using builders then you can use the `NoOpBuilderProv
 [[mapping-with-constructors]]
 === Using Constructors
 
-MapStruct also support using constructors for mapping target types.
+MapStruct supports using constructors for mapping target types.
 When doing a mapping MapStruct checks if there is a builder for the type being mapped.
 If there is no builder, then MapStruct looks for a single accessible constructor.
-WHen there are multiple constructors then the following is done to pick one that should be used:
+When there are multiple constructors then the following is done to pick the one which should be used:
 
-* If a parameterless constructor exists then it would be used to construct the object and the others would be ignored
-* If there are multiple constructors then the one annotated with annotation named `@Default` (from any package) would be used
+* If a parameterless constructor exists then it would be used to construct the object, and the others will be ignored
+* If there are multiple constructors then the one annotated with annotation named `@Default` (from any package) will be used
 
+When using a constructor then the names of the parameters of the constructor will be used as the target properties.
+When the constructor has an annotation named `@ConstructorProperties` (from any package) then this will be used to get the names of the parameters.
 
-When using a constructor then the names of the parameters of the constructor would be used as the target properties.
-If the constructor is annoated with an annotation named `@ConstructorProperties` (from any package) then this would be used to get the names of the parameters.
+[NOTE]
+====
+When an `@ObjectFactory` exists for a given target then that will have a precedence.
```

Comment:
..

When an object factory method or a method annotated with `@ObjectFactory` exist, it will take precedence over any constructor defined in the target.